Testscripts/Linux/channel_change.sh

```diff
@@ -175,5 +194,9 @@ function Main() {
 
 # main body
 Main
-SetTestStateCompleted
+if [ $FailedCount == 0 ]; then
```

Comment:
better to have an initial value for FailedCount, if no else block to enter, it will have syntax error.

Reply:
Move the variable initialization out of Main. Fixed in 56e05a9ae33d54c5d10114fd7fafcc8ac7eb5f83